front/app/containers/Admin/pagesAndMenu/containers/GenericHeroBannerForm/HeaderImageDropzone.tsx

```diff
@@ -85,18 +97,50 @@ const HeaderImageDropzone = ({
   const getImagePreviewRatio = () => {
     const layoutHeightOnDevice =
       homepageBannerLayoutHeights[layout][previewDevice];
-    const standardWidthPerDeviceType: { [key in TPreviewDevice]: number } = {
-      desktop: 1530,
-      tablet: 768,
-      phone: 375,
+    // Using the widths as we define them for our breakpoints.
+    const viewportWidthsKey = {
+      phone: 'phone' as const,
+      tablet: 'tablet' as const,
+      desktop: 'smallDesktop' as const,
+    }[previewDevice];
+    const standardDeviceWidth = viewportWidths[viewportWidthsKey];
+    const ratio = layoutHeightOnDevice / standardDeviceWidth;
+
+    const ratioPerLayoutPerDevice: {
+      [key in THomepageBannerLayout]: {
+        [key in TPreviewDevice]: number;
+      };
+    } = {
+      // For all layouts except fixed-ratio the fixed height in their styles
+      // Determine behavior.
+      full_width_banner_layout: {
+        phone: ratio,
+        tablet: ratio,
+        desktop: ratio,
+      },
+      two_column_layout: {
+        phone: ratio,
+        // On bigger screens, the image is only half the screen's width
+        // So the width gets divided by two (or the equivalent of twice the ratio).
+        tablet: 2 * ratio,
+        desktop: 2 * ratio,
+      },
+      two_row_layout: {
+        phone: ratio,
+        tablet: ratio,
+        desktop: ratio,
+      },
+      fixed_ratio_layout: {
+        // With our min-height of 225px, we assume a phone screenwidth of
+        // 450px for the sake of simplicity. Resulting in 1 / 2 aspect-ratio.
+        // Average phone screen width is slightly narrower though.
+        phone: 1 / 2,
+        tablet: 1 / 3,
```

Comment:
This PR fixes the divergence between citizen and admin, but also it introduces another potential divergence in the future if we don't tie the [real ratio](https://github.com/CitizenLabDotCo/citizenlab/blob/4d296d292d005050f4080ffa6e8740ad0719cbf3/front/app/components/LandingPages/citizen/FixedRatioBannerLayout.ts#L24) and the ratio used here.

I would consider moving it to a constant as we did for the project/folder header and card.